backend/src/main/scala/cromwell/backend/WorkflowBackendActor.scala

```diff
@@ -29,6 +30,10 @@ object WorkflowBackendActor {
   sealed trait AbortResponse extends WorkflowBackendActorResponse
   case class AbortSucceeded(jobKey: BackendJobDescriptorKey) extends AbortResponse
   case class AbortFailed(jobKey: BackendJobDescriptorKey, throwable: Throwable) extends AbortResponse
+
+  sealed trait ValidationResponse extends WorkflowBackendActorResponse
```

Comment:
Re this and a comment further down, I'm about to commit my FSM for ShadowWorkflowActor which builds on this. Hopefully you'll see how -

- WorkflowBackendActors start up immediately, and in the preStart will begin their validation (which might be slow if they need to make remote FS calls, but usually not)
- On `ValidationSuccess` or `ValidationFailed`, report back to the WorkflowActor which created them. The `ShadowWorkflowActor` aggregates the responses asynchronously and proceeds once it gets something back from every backend